docker-compose.yml

```diff
@@ -1,9 +1,35 @@
 version: "3.3"
 services:
+
   linchpino-core:
     image: ghcr.io/linchpino/linchpino-core:development
     container_name: linchpino-core
     restart: unless-stopped
     ports:
-      - "8080:8080"
+      - "8082:8080"
+    command: "mvn spring-boot:run -Dspring-boot.run.jvmArguments='-DPOSTGRES_USERNAME=linchpinodbuser -DPOSTGRES_PASSWORD=Ps@135792468 -DPOSTGRES_URL=jdbc:postgresql://api-dev.linchpino.com:5432/linchpinocore'"
```

Comment:
I suggest you add each parameter in a new line by "\" to increase code readability